doc-content/enterprise-only/project-data/test-designer-delete-test-proc.adoc

```diff
@@ -1,10 +1,10 @@
-[id='preview-editor-delete-test-proc']
+[id='test-designer-delete-test-proc']
 = Deleting a test scenario
 
-You can delete existing test scenarios which were created using the *Test Scenario (Preview)* editor.
+You can delete existing test scenarios which were created using the *Test Scenario* designer.
```

Comment:
Minor suggestion - replace "which" with "that":

s/You can delete existing test scenarios which were created using the *Test Scenario* designer./You can delete existing test scenarios that were created using the *Test Scenario* designer.

Reply:
Changed as suggested.